common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -44,3 +51,109 @@ innovation_war_camels = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+###Tech
+##Tribal
+
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_sea_locked
+	icon = @sea_faring
+	
+	potential = {
+		NOT = {
+			this.culture_group = culture_group:vrykul_group
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -0.8
+		garrison_size = -0.2
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+innovation_kyparite_sap = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_kyparite_regions
+	icon = @kyparite_sap
+
+	modifier = {
+		garrison_size = 0.02
+		skirmishers_damage_mult = 0.15
+		skirmishers_toughness_mult = 0.1
+		monthly_prestige_gain_mult = 0.02
+		monthly_piety_gain_mult = 0.02
```

Comment:
Why do you add these small modifiers that barely affect?

Reply:
![image](https://user-images.githubusercontent.com/74790101/113435998-e6773a80-93b1-11eb-9bf5-3f7aa25019f6.png)

These changes okay?

Reply:
> These changes okay?

Ok.